src/com/dynatrace/loadrunner/Main.java

```diff
@@ -1,59 +1,202 @@
-
-        List<ScriptFile> currentFileList = null;
-        for (int i = 2; i < args.length; i++) {
-            if (args[i].equals("-h")) {
-                hFiles = new ArrayList<>();
-                currentFileList = hFiles;
-            } else if (args[i].equals("-c")) {
-                cFiles = new ArrayList<>();
-                currentFileList = cFiles;
-            } else if (currentFileList != null) {
-                File file = new File(args[i]);
-                if (!file.exists()) {
-                    System.out.println("Could not load " + args[i]);
-                    return;
-                }
-                currentFileList.add(new ScriptFile(file));
-            } else {
-                printUsage();
-                return;
-            }
-        }
-
-        System.out.println("Mode: " + mode);
-
-        LoadRunnerConverter converter = new LoadRunnerConverter();
-        converter.configureConverter(mode, hFiles, cFiles);
-        converter.convertFiles();
-    }
-}
+	private static List<ScriptFile> headers = new ArrayList<>();
+	private static List<ScriptFile> body = new ArrayList<>();
+	private static String scriptName = "";
+	private static Boolean mode;
+	private static boolean cEngine = true;
+	private static final String VERSION = "Dt-LoadRunner-request-tagging version 2.0.1";
+
+	public static void showUsages() {
+		
+		System.out.println("The LoadRunner Request Tagging tool uses the following syntax: \n"
+				+ "java -jar Dt-LoadRunner-request-tagging.jar <mode> <path parameter> <optional parameters> \n"
+				+ "\n"
+				+ "mode:\n"
+				+ "	insert: to add the Dynatrace HTTP header to the selected LoadRunner scripts, type insert\n"
+				+ "	delete: to remove all modifications previously made by the LoadRunner Request Tagging tool, type delete\n"
+				+ "path parameter:\n"
+				+ "	Pick either -path or -body and -header\n"
+				+ "	-path <filepath>: use to scan all directories and subdirectories for script file's and insert/delete script in them\n"
+				+ "	-body <files> -header <files>: use to select exactly in which header and body files should be processed, the file seperator is & between files\n"
+				+ "optional parameteer:\n"
+				+ "	-LSN <value>: sets load script name to value passed after -LSN. If skipped, the script name will be taken from *.usr file\n"
+				+ "	-c: sets C as scripting language used (default)\n"
+				+ "	-js: sets JavaScript as scripting language\n"
+				+ "	-help: prints usage\n");
+	}
+
+	public static void main(String[] args) {
+
+		System.out.println(VERSION + "\n");
+
+		if (args.length == 0) {
+			showUsages();
+			return;
+		}
+
+		CommandLineParser parser = new CommandLineParser(args);
+
+		if (parser.arguments.containsKey("-help") || parser.arguments.containsKey("help")) {
+			showUsages();
+			return;
+		}
+
+		for (Entry<String, String> pair : parser.arguments.entrySet()) {
+			String key = pair.getKey().toLowerCase();
+			if (key.equals("insert")) {
```

Comment:
define all the param strings as `private static final String`s, and use `switch(key) { case ...; default: ... }` to recognize them